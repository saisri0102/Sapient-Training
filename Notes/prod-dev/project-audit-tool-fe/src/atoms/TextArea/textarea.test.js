import { render, screen } from "@testing-library/react";

import userEvent from "@testing-library/user-event";

test("Comment Box", () => {
  const wrapper = render(
    <textarea
      type="text"
      className="comment-box"
      id="comm"
      placeholder="Write a comment..."
      rows="3"
      col="50"
    ></textarea>
  );

  expect(wrapper).toBeDefined();

  expect(wrapper).toMatchSnapshot();
});
