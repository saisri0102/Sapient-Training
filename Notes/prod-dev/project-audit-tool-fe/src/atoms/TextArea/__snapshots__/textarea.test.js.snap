// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment Box 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <textarea
        class="comment-box"
        col="50"
        id="comm"
        placeholder="Write a comment..."
        rows="3"
        type="text"
      />
    </div>
  </body>,
  "container": <div>
    <textarea
      class="comment-box"
      col="50"
      id="comm"
      placeholder="Write a comment..."
      rows="3"
      type="text"
    />
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
